src/core/support/slice_buffer.c

```diff
@@ -117,7 +117,7 @@ void gpr_slice_buffer_add(gpr_slice_buffer *sb, gpr_slice s) {
                s.data.inlined.bytes, s.data.inlined.length);
         back->data.inlined.length += s.data.inlined.length;
       } else {
-        size_t cp1 = GPR_SLICE_INLINED_SIZE - back->data.inlined.length;
+		gpr_uint32 cp1 = GPR_SLICE_INLINED_SIZE - back->data.inlined.length;
```

Comment:
Also, your editor is set to insert tabulation instead of spaces. You can use an [EditorConfig](http://editorconfig.org/#download) plugin to help you with grpc's formatting.